owncloudApp/src/main/java/com/owncloud/android/presentation/transfers/TransferListFragment.kt

```diff
@@ -109,6 +111,11 @@ class TransferListFragment : Fragment() {
                 transfersAdapter.updateTransferProgress(workInfo)
             }
         }
+
+        collectLatestLifecycleFlow(transfersViewModel.spaces) {
+            transfersViewModel.transfersListLiveData.value?.let { transfers -> setData(transfers) }
+        }
```

Comment:
Not sure about this, it does nothing at all. It sets data that was already there, since it takes the transfers from the viewmodel. Am I missing something?

Reply:
Yes, this invokes the `setData` method of the Fragment, not the one of the Adapter. Inside that method, the spaces  are taken from the ViewModel and sent to the Adapter. These lines are necessary just in case the transfers are retrieved before the spaces are 👍